CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
 
 
+## Unreleased
+
+### Deprecated
+
+- The `debug_plugins` option for `PluginClient` is deprecated and will be removed in 2.0.
+
```

Comment:
yep. and maybe point to one place where we do that ourselves as an illustration. thats a bit verbose for a changelog but i would prefer that over writing extensive upgrade guides...